patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -9,35 +9,37 @@
 +    protected java.util.HashMap<Integer, Boolean> spawnForcedMap = new java.util.HashMap<Integer, Boolean>();
 +    public float eyeHeight = this.getDefaultEyeHeight();
 +    public static final net.minecraft.entity.ai.attributes.IAttribute REACH_DISTANCE = new net.minecraft.entity.ai.attributes.RangedAttribute(null, "generic.reachDistance", 5.0D, 0.0D, 1024.0D).func_111112_a(true);
++    public static final net.minecraft.entity.ai.attributes.IAttribute SWIM_SPEED = new net.minecraft.entity.ai.attributes.RangedAttribute(null, "forge.swimSpeed", 1.0D, 0.0D, 1024.0D).func_111112_a(true);
```

Comment:
Why not have this in entitylivingbase? Non players can be affected by this too, I'm pretty sure

Reply:
I honestly hadn't considered that until after I submitted the pull request. I think you're right that there's no reason NOT to have it there as the code itself doesn't add any additional overhead for it's execution.